pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -208,7 +208,7 @@ func init() {
 				},
 				// DeploymentController.podClient
 				{
-					Verbs:     sets.NewString("get", "list", "create", "watch", "delete", "update"),
+					Verbs:     sets.NewString("get", "list", "watch"),
```

Comment:
@deads2k down here we have:

```
 				{
 					Verbs:     sets.NewString("create", "update", "patch"),
 					Resources: sets.NewString("events"),
 				},
```

and this is copied in other policies... do we really need PATCH and UPDATE for events?